src/components/App.tsx

```diff
@@ -343,7 +343,7 @@ class App extends React.Component<AppProps, AppState> {
           style={{
             width: canvasDOMWidth,
             height: canvasDOMHeight,
-            cursor: "grabbing",
+            cursor: CURSOR_TYPE.POINTER,
```

Comment:
We should use `grab` instead of `pointer` for view mode I believe.

Reply:
switched it back to grab